changelog.md

```diff
@@ -2,6 +2,30 @@
 
 ## [Unreleased]
 
+## [0.3.0] - 2023-01-31
+
+### Added
+
+- Basic support for auto-generated insert procedures for tables.
+  ([#261](https://github.com/hasura/ndc-postgres/pull/261))
+- Support for composite types in inputs and outputs. Transformations are not yet
+  supported.
```

Comment:
'Transformations'? Perhaps you meant:

```suggestion
- Support for composite types in inputs and outputs. Automatic introspection
  is not yet supported.
```

Reply:
I meant that we don't provide any operators on these composite types. I will add both.

```suggestion
- Support for composite types in inputs and outputs. Right now, no operators are
  provided for composite types, and there is no automatic introspection.
```

Reply:
Hm.. I still wonder if it's relevant to say that, since the ndc spec does not allow for any operators on things that are not scalar types.

Wording it like this suggests this is just something that has not been implemented in ndc-postgres yet.

Reply:
I am personally OK with that. I think it's more important to state that we do or don't do something than _why_.

If someone cares, they will ask!